01_fibonacci/fibi_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Let's test `fibonacci ` function as well (unit tests ;) ) and include some corner cases:
```go
n :=  0
n := 1
n := -1
n := // find a very large number that causes a stackoverflow and comment....